oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java

```diff
@@ -251,19 +321,64 @@ public GoogleCredentials(AccessToken accessToken) {
     this(accessToken, null);
   }
 
+  /**
+   * Constructor with using builder All the fields comes explicitly from builder.
+   *
+   * @param builder an instance of a builder
+   */
   protected GoogleCredentials(Builder builder) {
-    this(builder.getAccessToken(), builder.getQuotaProjectId());
+    super(builder.getAccessToken());
+    this.quotaProjectId = builder.getQuotaProjectId();
+    this.universeDomain = builder.getUniverseDomain();
   }
 
   /**
-   * Constructor with explicit access token and refresh times
+   * Constructor with explicit access token and refresh times.
+   *
+   * <p>Deprecated, please use the builder constructor whenever possible.
```

Comment:
Let's link it here (and same everywhere else where this applies)

Reply:
Agree, done